src/app/shared/models/device/device.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface Device {
```

Comment:
I think it should be call like: DotDevice

Reply:
Yup